Makefile

```diff
@@ -40,5 +40,14 @@ migrations: ## Make migrations to the wagtail bakery site
 migrate: ## Migrate the wagtail bakery site migrations
 	docker-compose exec web python manage.py migrate
 
-test: ## Run all wagtail tests or pass in a file with `make test file=wagtail.admin.tests.test_name.py`
```

Comment:
The runner in Wagtail uses the Django test runner, so it expects a python module name. Hence, no `.py` should be used at the end.